reconstruction.py

```diff
@@ -892,7 +892,7 @@ def reconstruct(self,evrange=(-1,-1,-1)):
             postfix = 'mid.gz'
         options.tmpname = "%s/%s%05d.%s" % (tmpdir,prefix,int(options.run),postfix)
     else:
-        if options.rawdata_tier == 'root':
+        if options.rawdata_tier == 'root' or options.rawdata_tier == 'h5':
             file_url = sw.swift_root_file(options.tag, int(options.run))
```

Comment:
Why was this added? swift_root_file returns a .root address which cannot work for h5, right?. If you are not downloading the h5 files, I would add a if statement and a print with a sys.exit() in case you reach this part of the code and it seems you need to download a h5 file. Now h5 is not downloadable and we will add this feature when/if it will be necessary. Does this make sense?